pkg/server/app/app.go

```diff
@@ -328,6 +330,33 @@ func checkForProtectedEnvVars(evsNames []string) error {
 	return nil
 }
 
+func (ops *AppOperations) List(user *storage.User) ([]*List, error) {
+	teams, err := ops.tops.ListByUser(user.Email)
+	if err != nil {
+		log.Errorf("ErrNotFound")
```

Comment:
It'd be nice to use a variable instead of writing the string `ErrNotFound` on every `log.Errorf` calls.
And it seems the variable is already defined in `errors.ErrNotFound`. Does this makes sense @drgarcia1986?